.gitignore

```diff
@@ -15,3 +15,4 @@ bin/yum-2
 bin/yum-3
 *.cmake
 *CMakeCache*
+.vscode/
```

Comment:
Please don't add this here, add it to your global .gitignore. Thanks.

Reply:
Do you want me to back out this particular commit?

Reply:
Yes please.

Reply:
This has been completed.